README.md

```diff
@@ -1,15 +1,20 @@
+# Hello-microverse
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
```

Comment:
- [ ] As asked by the previous reviewer, please remove the `Project Name` keyword, since it is kind of confusing when looking at the readme 
![image](https://user-images.githubusercontent.com/62705947/185175016-68d96828-9232-4e08-8eca-6b22350afcad.png)